package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Protocol definition for generic messages.",
+  "license": "GPL-3.0",
+  "main": "./proto",
+  "name": "wire-webapp-protocol-message",
```

Comment:
I can see it's alphabetical.. but it still feels that the name should be first maybe?!